fh_fablib/pre-commit-defaults-with-formatting.yaml

```
@@ -0,0 +1,54 @@
+repos:
+    - repo: local
+      hooks:
+          - id: django-check
+            name: django check
+            entry: sh -c 'venv/bin/python manage.py check'
+            pass_filenames: false
+            language: system
+            always_run: true
+    - repo: https://github.com/asottile/pyupgrade
+      rev: v2.29.1
+      hooks:
+          - id: pyupgrade
+            args: [--py36-plus]
+    - repo: https://github.com/psf/black
+      rev: 21.10b0
+      hooks:
+          - id: black
+    - repo: https://github.com/pycqa/isort
+      rev: 5.10.1
+      hooks:
+          - id: isort
+            args:
+                [
+                    --virtual-env=venv,
+                    --profile=black,
+                    --lines-after-imports=2,
+                    --combine-as,
+                ]
+    - repo: https://github.com/pycqa/flake8
+      rev: 4.0.1
+      hooks:
+          - id: flake8
+    - repo: https://github.com/pre-commit/mirrors-prettier
+      rev: v2.4.1
+      hooks:
+          - id: prettier
+            args: [--list-different]
+            files: "###FRONTEND_STATIC_PATTERN###"
+            types_or: [css, javascript]
```

```suggestion
            types_or: [css, javascript, scss]
```

Ich bin nicht sicher, aber vermute, dass es SCSS explizit braucht:
https://github.com/pre-commit/identify/blob/master/identify/extensions.py#L168